Dangerfile

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+# Runs rails_best_practices on modified and added files in the PR
+rails_best_practices.lint
+
+# Runs Rubocop and submit comments on modified and added files
+rubocop.lint(inline_comment: true, force_exclusion: true)
+
+# Runs Brakeman on modified and added files
+brakeman.run
```

Comment:
Can we put the `inline_comment: true` to this lint and reek too @noor-ahmed 🤔

Reply:
@anduonghien As far as I have seen [danger-brakeman](https://github.com/nakamurau1/danger-brakeman) and [danger-reek](https://github.com/blooper05/danger-reek) doesn't have that support yet. Can you share some insight if you have ideas?

Reply:
Well I never use Danger before 😂 so just asking there @rafayetn 😄 so the `brakeman.run` won't add the comment to the PR 🤔

Reply:
Yes, it will add comments but it won't add comments like [danger-rubocop](https://github.com/ashfurrow/danger-rubocop) at [inline of the diffs](https://user-images.githubusercontent.com/14325963/90365448-98623080-e087-11ea-91c0-4cf56906f7a8.png).